src/App.vue

```diff
@@ -40,6 +44,9 @@ const keyColors = ref([
 ]);
 const isomorphicVertical = ref(5);
 const isomorphicHorizontal = ref(1);
+const keyboardMapping = reactive<Map<string, number>>(new Map());
+mapWhiteAsdfBlackQwerty(keyColors.value, keyboardMapping);
+const keyboardMode = ref<"isomorphic" | "mapped">("isomorphic");
```

Comment:
I intended "mapped" to be a free mapping, but #182 would warrant a third mode that only includes canonical layouts built from key colors and thus has a small URL footprint.